docs/TODO

```diff
@@ -793,31 +793,23 @@ that doesn't exist on the server, just like --ftp-create-dirs.
  Add a way to check if the connection seems to be alive, to correspond to the
  SSL_peak() way we use with OpenSSL.
 
-15. WinSSL/SChannel
+15. Schannel
 
-15.1 Add support for client certificate authentication
+15.1 Extend support for client certificate authentication
 
- WinSSL/SChannel currently makes use of the OS-level system and user
- certificate and private key stores. This does not allow the application
- or the user to supply a custom client certificate using curl or libcurl.
-
- Therefore support for the existing -E/--cert and --key options should be
- implemented by supplying a custom certificate to the SChannel APIs, see:
+ The existing support for the -E/--cert and --key options could be
+ extended by supplying a custom certificate and key in PEM format, see:
  - Getting a Certificate for Schannel
    https://msdn.microsoft.com/en-us/library/windows/desktop/aa375447.aspx
 
-15.3 Add support for the --ciphers option
-
- The cipher suites used by WinSSL/SChannel are configured on an OS-level
- instead of an application-level. This does not allow the application or
- the user to customize the configured cipher suites using curl or libcurl.
+15.2 Extend support for the --ciphers option
 
- Therefore support for the existing --ciphers option should be implemented
- by mapping the OpenSSL/GnuTLS cipher suites to the SChannel APIs, see
+ The existing support for the --ciphers option could be extended
+ by mapping the OpenSSL/GnuTLS cipher suites to the Schannel APIs, see
```

Comment:
I don't know why we have this in the TODO. We support --ciphers but the problem is there's no way to implement specifying individual ciphers like we can with OpenSSL. We can specify groups of ciphers, that might be as good as it gets. This feels more like a known bugs entry to me. We just can't do it, or maybe we can but it's going to be reinventing the wheel.

Reply:
I guess we would need to create a mapping list from cipher suites to a combination of algorithms. At least this is what I mean here and I think it is an enhancement rather than a known bug.